frontend/locales/fi.json

```diff
@@ -201,7 +201,8 @@
     "scheduled_time": "Aloitusaika",
     "no_scheduled_time": "Aloitusaikaa ei ole määritetty",
     "ongoing": "Käynnissä:",
-    "no_matches": "Ei otteluita"
+    "no_matches": "Ei otteluita",
+    "team_name": "Joukkueen nimet"

```

Comment:
I dont know the context but "Joukkueen nimet" is "Team's names"

"Team name" would be "Joukkueen nimi"